sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosBridgeInternal.java

```diff
@@ -51,6 +51,10 @@ public static CosmosAsyncContainer getCosmosContainerWithNewClient(CosmosAsyncCo
     public static AsyncDocumentClient getContextClient(CosmosAsyncDatabase database) {
         return database.getClient().getContextClient();
     }
+    @Warning(value = INTERNAL_USE_ONLY_WARNING)
```

Comment:
No new methods here - use Implementationbridgehelpers pattern instead

Reply:
Changed to use ImplementationBridgeHelpers